Omulosi/iReporter
# #162365810 Add coverage and maintainability badges

#### What does this PR do?
Adds coverage and maintainability badges to the README.md file
#### Description of Task to be completed?
Copy the markdown code from codecav and codelimate to to the README.md file to embed status badges
#### How should this be manually tested?
Open the Omulosi/iReporter repositoery and navigate to the ch-add-coverage-badge-162365810. The README.md file in this file should display the status badges.
#### What are the relevant pivotal tracker stories?
#162365810
